PeerioTechnologies/peerio-mobile
# fix: change delete to remove inline file [branch 8303]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8303/mobile-change-delete-to-remove-when-user-is-not-owner
https://github.com/PeerioTechnologies/peerio-icebear/pull/249
#### Testing instructions  
Inline files:
case 1: check copy for when user is the file owner
case 2: check copy for when user is NOT the file owner

Note: Check that both cases remain unchanged for a file in files tab.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
